python/Ganga/Core/GangaRepository/__init__.py

```diff
@@ -20,3 +20,10 @@ def getRegistries():
 
 def getRegistry(name):
     return allRegistries[name]
+
+def getRegistrySlice(name):
+    return allRegistries[name].getSlice()
+
+def getRegistryProxy(name):
+    return allRegistries[name].getProxy()
+
```

Comment:
If we're introducing new functions then they need full docstrings (preferably in the Google style see [here](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) and [here](http://www.sphinx-doc.org/en/stable/ext/napoleon.html#module-sphinx.ext.napoleon)).